pkg/controller.v1beta1/experiment/util/status_util.go

```diff
@@ -177,8 +186,9 @@ func getObjectiveMetricValue(trial trialsv1beta1.Trial) string {
 // UpdateExperimentStatusCondition updates the experiment status.
 func UpdateExperimentStatusCondition(collector *ExperimentsCollector, instance *experimentsv1beta1.Experiment, isObjectiveGoalReached bool, getSuggestionDone bool) {
 	logger := log.WithValues("Experiment", types.NamespacedName{Name: instance.GetName(), Namespace: instance.GetNamespace()})
-	completedTrialsCount := instance.Status.TrialsSucceeded + instance.Status.TrialsFailed + instance.Status.TrialsKilled + instance.Status.TrialsEarlyStopped
-	failedTrialsCount := instance.Status.TrialsFailed
+	completedTrialsCount :=
+		instance.Status.TrialsSucceeded + instance.Status.TrialsFailed + instance.Status.TrialsKilled + instance.Status.TrialsEarlyStopped + instance.Status.TrialMetricsUnavailable
+	failedTrialsCount := instance.Status.TrialsFailed + instance.Status.TrialMetricsUnavailable
```

Comment:
Discussion point:
Is TrialMetricsUnavailable state a failed state for a user?

Reply:
I think that the TrialMetricsUnavailable state is a failed Trial for a user since katib can not use that Trial result to optimize hyperparameters even though the training is successful if the metrics-collector container fails to collect metrics.

What do you think? @johnugeorge

Reply:
I think so since it is a unrecoverable as well. We have to include this in docs as well to explain "Failed" trial"

Reply:
It makes sense.
As I can see katib docs, we don't have any sections to describe Experiment status, so It might be better to explain Experiment status in [this](https://www.kubeflow.org/docs/components/katib/overview/#experiment) paragraph describing Experiment.

Does it sound good to you? @johnugeorge

Reply:
Sure. We will wait for a day to get other reviews as well.